README.md

```diff
@@ -33,6 +33,13 @@ and [contributing guidelines](CONTRIBUTING.md).
 
 1. FerretDB uses the same protocol error names and codes, but the exact error messages may be different in some cases.
 
+2. Collection names restrictions:
+
+* collection name cannot start with the reserved prefix `_ferretdb_`.
+* collection name must not include non-latin letters, spaces, dots, dollars or dashes.
+* collection name length must be less or equal than 119 symbols.
```

Comment:
120, actually. The length should be in the range [1-120] inclusively

Reply:
> 120, actually. The length should be in the range [1-120] inclusively

gotcha. I thought what a strange number 119. fixed.